docs/cri/config.md

```diff
@@ -228,7 +228,8 @@ version = 2
   # 'plugins."io.containerd.grpc.v1.cri".containerd' contains config related to containerd
   [plugins."io.containerd.grpc.v1.cri".containerd]
 
-    # snapshotter is the snapshotter used by containerd.
+    # snapshotter is the default snapshotter used by containerd.
```

Comment:
s/snapshotter is the default snapshotter used by containerd./snapshotter is the default snapshotter used by containerd for all runtimes, if not overridden by a runtime's snapshotter config./

Reply:
Replaced